### PR TITLE
feat(app): adjust status query param for subscription-status

### PR DESCRIPTION
**Pull request by Phil91**
## Description

* added query param `name` to endpoint `GET: /api/apps/subscribed/subscription-status` to filter for app names
* added query param `name` to endpoint `GET: /api/services/subscribed/subscription-status` to filter for service names
* changed query param `statusId` to `status` of endpoint `GET: /api/apps/subscribed/subscription-status`

## Why

To be able to search for specific apps

## Issue

Refs: #592

## Checklist

- [x] I have followed the [contributing guidelines](https://github.com/eclipse-tractusx/portal-assets/blob/main/docs/developer/Technical%20Documentation/Dev%20Process/How%20to%20contribute.md#commit-and-pr-guidelines)
- [x] I have performed [IP checks](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-04#checking-libraries-using-the-eclipse-dash-license-tool) for added or updated 3rd party libraries
- [x] I have created and linked IP issues or requested their creation by a committer
- [x] I have performed a self-review of my own code
- [x] I have successfully tested my changes locally
- [x] I have added tests that prove my changes work
- [x] I have checked that new and existing tests pass locally with my changes
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/marketplace/Apps.Service/Controllers/AppsController.cs [hunk 1]
```diff
@@ -158,7 +158,8 @@ public async Task<IActionResult> RemoveFavouriteAppForCurrentUserAsync([FromRout
     /// <remarks>Example: GET: /api/apps/subscribed/subscription-status</remarks>
     /// <param name="page">The page that should be displayed</param>
     /// <param name="size">The size per page of elements that should be returned</param>
-    /// <param name="statusId">Filter for the offer subscription status. If not set all elements will be returned</param>
+    /// <param name="status">Filter for the offer subscription status. If not set all elements will be returned</param>
```
Comment (evegufy) [inline src/marketplace/Apps.Service/Controllers/AppsController.cs:161]:
```suggestion
    /// <param name="status">Filter for the offer subscription status. If not set, all elements will be returned</param>
```

File: src/marketplace/Services.Service/Controllers/ServicesController.cs [hunk 2]
```diff
@@ -297,7 +297,8 @@ public Task<SubscriberSubscriptionDetailData> GetSubscriptionDetailForSubscriber
     /// <remarks>Example: GET: /api/services/subscribed/subscription-status</remarks>
     /// <param name="page">The page that should be displayed</param>
     /// <param name="size">The size per page of elements that should be returned</param>
-    /// <param name="statusId">Filter for the offer subscription status. If not set all elements will be returned</param>
+    /// <param name="status">Filter for the offer subscription status. If not set all elements will be returned</param>
```
Comment (evegufy) [inline src/marketplace/Services.Service/Controllers/ServicesController.cs:300]:
```suggestion
    /// <param name="status">Filter for the offer subscription status. If not set, all elements will be returned</param>
```

**Review by evegufy — CHANGES_REQUESTED** [review]

**Comment by evegufy** [comment]
@MaximilianHauer @Phil91 is the milestone 24.12 correct for https://github.com/eclipse-tractusx/portal-backend/issues/592?

File: src/marketplace/Services.Service/Controllers/ServicesController.cs (hunk 2 above)
Comment (Phil91) [reply src/marketplace/Services.Service/Controllers/ServicesController.cs]: adjusted

**Comment by Phil91** [comment]
> @MaximilianHauer @Phil91 is the milestone 24.12 correct for #592?

@evegufy in my opinion it isn't

**Review by ntruchsess — APPROVED** [review]

**Pull request merged**